lib/realtime/syn_handler.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Realtime.SynHandler do
+  @moduledoc """
+  Custom defined Syn's callbacks
+  """
+  require Logger
+  alias RealtimeWeb.Endpoint
+
+  def on_process_unregistered(mod, name, _pid, _meta, reason) do
+    Logger.warn("#{mod} terminated: #{inspect(name)} #{node()}")
+    topic = topic(mod)
+
+    if reason != :syn_conflict_resolution do
+      Endpoint.local_broadcast(topic <> ":" <> name, topic <> "_down", nil)
```

Comment:
The `realtime_channel` reconnects after receiving the `postgres_cdc_down` event. However, after this change, the event will be different: `postgres_cdc_rls_down` for RLS and "postgres_cdc_stream_down" for stream

Reply:
btw, `Realtime.Tenants.Connect` should also handle tenant termination, because it needs to unregister the syn record and close the connection to db

Reply:
will change 👍

Reply:
posted a question in slack for clarification regarding this change

Reply:
clarification made, will handle here as initially planned